include/boost/gil/extension/numeric/affine.hpp

```diff
@@ -108,6 +108,36 @@ boost::gil::matrix3x2<T> inverse(boost::gil::matrix3x2<T> m)
     return res;
 }
 
+//feature
+//rotate an image from its center point for -180 deg <= theta <= 180 deg
+//using consecutive affine transformations
+//As top-left corner of source image is taken as origin, idea is to rotate the source image about the origin and then translate the new image
+//according to the mentioned rotation angle, and to re-center and fit the image in the source dimensions 
+// preferably use type double for less precision errors
+template<typename T, typename F> boost::gil::matrix3x2<F> center_rotate(boost::gil::point<T> dims,F rads)
+{   
+    while(rads < F(-M_PI)) rads = rads + F(M_PI);
+    while(rads > F(M_PI)) rads = rads - F(M_PI);
```

Comment:
`rads = ...` in new line

Reply:
I have cleaned it up 

https://github.com/boostorg/gil/pull/565/files#diff-89d5dde7ab09b71c771dca84e09c6c12bcdf7a5466eb3b65beff1882e77ba8f2R127-R135